src/components/UI/ButtonStyled/Button.styled.js

```diff
@@ -16,6 +16,7 @@ export const Button = styled.button`
 			border: none;
 			position: relative;
 			margin-top: 20px;
+			font-family: 'Courier New', Courier, monospace;
```

Comment:
You use the same font all over your app, you should add it to the global styles instead of defining it over and over in every component (DRY)